packages/mobile/locales/es-419/accountScreen10.json

```
@@ -4,21 +4,21 @@
   "languageSettings": "Idioma",
   "localCurrencySetting": "Moneda",
   "licenses": "Licencias",
-  "removeAccountTitle": "Reiniciar Valora",
-  "removeAccountDetails": "Reiniciar borrará tu cuenta de este dispositivo. Tu cuenta mantendrá el dinero que tenga, pero solo será recuperable con tu Clave de Cuenta",
+  "removeAccountTitle": "Reinicializar Valora",
+  "removeAccountDetails": "Al reiniciar se borrará tu cuenta de este dispositivo. Tu cuenta mantendrá los fondos que tenga, pero solo la podrás recuperar con tu Clave de Cuenta.",
```

```suggestion
  "removeAccountDetails": "Al reinicializar se borrará tu cuenta de este dispositivo. Tu cuenta mantendrá los fondos que tenga, pero solo la podrás recuperar con tu Clave de Cuenta.",
```

Should this also be `reinicializar`?